.changelog/_unreleased.yml

```diff
@@ -0,0 +1,5 @@
+changes:
+- type: fix|docs
```

Comment:
This needs to be just `fix` or `docs` 🙂

Reply:
This has been fixed now. Happy to contribute :)